File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromServer/UpdateMessage.java
package org.demoth.aworlds.server2.api.messaging.fromServer;

import org.demoth.aworlds.server2.api.messaging.Message;

import java.util.List;

public class UpdateMessage extends Message {
    public List<Message> updates;

    public UpdateMessage() {
    }

    public UpdateMessage(List<Message> updates) {
        this.updates = updates;
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromServer/UpdateMessage.kt
package org.demoth.aworlds.server2.api.messaging.fromServer

import org.demoth.aworlds.server2.api.messaging.Message

class UpdateMessage(var updates: List<Message>) : Message() {

    override fun toString(): String {
        return "UpdateMessage{" +
                "updates=" + updates +
                '}'.toString()
    }
}
